packages/maskbook/src/components/InjectedComponents/SetupGuide.tsx

```diff
@@ -271,6 +350,8 @@ function FindUsername({ username, onConnect, onDone, onClose, onUsernameChange =
     const { t } = useI18N()
     const ui = getActivatedUI()
 
+    const isMobile = useMatchXS()
```

Comment:
```suggestion
    const xsMatched = useMatchXS()
```
This hook only checks the screen width, small screen is not always mobile.